app/src/main/java/org/mozilla/tv/firefox/channels/ChannelTile.kt

```diff
@@ -37,7 +37,9 @@ data class ChannelTile(
             TileSource.BUNDLED, TileSource.CUSTOM ->
                 context.resources.getString(R.string.pinned_tiles_channel_remove_title, title)
             TileSource.POCKET -> throw NotImplementedError("pocket shouldn't be able to remove tiles")
-            TileSource.TV_GUIDE -> "" // TODO in #2326
+            TileSource.NEWS -> "" // TODO in #2326
+            TileSource.SPORTS -> "" // TODO in #2326
+            TileSource.MUSIC -> "" // TODO in #2326
```

Comment:
It looks like these need to be added.  Can they be the same as for BUNDLED and CUSTOM?

Reply:
@brampitoyo Can the remove strings be unified for TV_GUIDE?